### ISSUE TITLE
what is the best approach to customize base url with subfolder?

### ISSUE THREAD

**Issue by xhe** [swagger-api/swagger-ui#1059]
It seemed Swagger will create ajax end point by adding url to base URL.
But what is the best approach to customize base URL?
For example:
Our api-docs is here: http://www.example.com/swagger/api-docs
And our api is v0/users, so the final URL for api should be http://www.example.com/swagger/v0/users
But when I test swagger, I noticed it is this way: http://www.example.com/v0/users
the subfolder is removed, just appending URL to root.
Can anyone help to point out what is best approach to customize base URL?
Thanks

**Comment by mohsen1** [comment]
Setting your `host` and `basePath` in the specs should fix this issue. You're probably using a relative `basePath`

**Comment by xhe** [comment]
Sorry, it does not work for me.
The URL for doc is: http://localhost:8080/api-docs
Then I retrieved all the listings of api, 
And then, I config swagUI as this:
  window.swaggerUi = new SwaggerUi({
        url: url,
        basePath: "swagger",
...
}
I want to have url endpoint to be http://localhost:8080/swagger/v0/users, but I found the api ajax url is:
http://localhost:8080/v0/users, the basePath is not inserted.
Any idea?
